### PR TITLE
Added a test for PR 6305

### PR DESCRIPTION
**Pull request by srwiseman**
@lilleyse, as discussed, I added a test for #6305.

Basically, I copied the "computes bounding sphere" test, but I changed it to give it a bad model uri. The test then expects the bounding sphere state to resolve to FAILED, which it does due to the fix in #6305.

Let me know if this works!

### CODE REVIEW

File: Specs/DataSources/ModelVisualizerSpec.js [hunk 1]
```diff
@@ -292,6 +292,32 @@ defineSuite([
...
     });
 
+    it('Fails bounding sphere when model fails to load.', function() {
+        var entityCollection = new EntityCollection();
+        visualizer = new ModelVisualizer(scene, entityCollection);
+
+        var time = JulianDate.now();
+        var testObject = entityCollection.getOrCreateEntity('test');
+        var model = new ModelGraphics();
+        console.log(model);
```
Comment (lilleyse) [inline Specs/DataSources/ModelVisualizerSpec.js]: Remove `console.log`.

File: Specs/DataSources/ModelVisualizerSpec.js [hunk 2]
```diff
@@ -292,6 +292,32 @@ defineSuite([
...
+        testObject.model = model;
+
+        testObject.position = new ConstantProperty(new Cartesian3(5678, 1234, 1101112));
+        model.uri = new ConstantProperty("/path/to/incorrect/file");
```
Comment (lilleyse) [inline Specs/DataSources/ModelVisualizerSpec.js]: Style: use single quotes:

`model.uri = new ConstantProperty('/path/to/incorrect/file');`

Comment (srwiseman) [reply Specs/DataSources/ModelVisualizerSpec.js]: Ah, must have left the console log in there when I was debugging! Sorry!

Will also fix the quotes.

**Comment by lilleyse** [comment]
Thanks @srwiseman

**Pull request merged**